omf/fileio/geoh5.py

```diff
@@ -305,6 +311,18 @@ def collect_attributes(
         return kwargs
 
 
+class KnownUnsupported(BaseConversion):
```

Comment:
interesting trick

Can you just explain me the logic? 
(maybe add it in the docstring)